packages/components/Button/.npmignore

```diff
@@ -0,0 +1,2 @@
+# Don't include test files in package
+__tests__/
```

Comment:
If we need this in every package, can we also update the component-generator later, if not in this PR? I know that isn't heavily used but I think some people may still try to use that as their "first new package" template

Reply:
yeah, that makes sense. I can also add it top level instead.